lib/ast-converter.js

```diff
@@ -954,8 +954,14 @@ module.exports = function(ast, extra) {
                 break;
 
             case SyntaxKind.VariableStatement:
+                var variableDeclarationType = "VariableDeclaration",
+                    variableIsAmbient = ts.isInAmbientContext(node);
+
+                if (variableIsAmbient) {
+                    variableDeclarationType = "TSAmbientVariableDefinition";
```

Comment:
You used definition for the the methods in #163 and I just followed that because that what I based this PR on. But I agree with you Declaration would probably be better in this case, will update.